tic_tac_toe/game.py

```diff
@@ -63,22 +81,66 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    diagonals = [ 
+                    ((2, 0), (1, 1), (0, 2)),
+                    ((0, 0), (1, 1), (2, 2))
+                ]
+    columns = [
+                    ((0, 0), (1, 0), (2, 0)),
+                    ((0, 1), (1, 1), (2, 1)),
+                    ((0, 2), (1, 2), (2, 2))
+              ]
+    rows = [         
+                    ((0, 0), (0, 1), (0, 2)),
+                    ((1, 0), (1, 1), (1, 2)),
+                    ((2, 0), (2, 1), (2, 2))
+           ]
+
+    for diagonal in diagonals:
+        if _is_winning_combination(board, diagonal, player): return player
+    for column in columns:
+        if _is_winning_combination(board, column, player): return player
+    for row in rows:
+        if _is_winning_combination(board, row, player): return player
+    return None
+
```

Comment:
Just a suggestion. You could have put all the combinations in one list then only had one for loop instead of 3. Even if you wanted 3 lists you could have one for loop by:

    for combinations in diagonals+columns+rows:
        if _is_winning_combination(board, combination, player):
            return player
 
Remember the mantra DYI. (Don't Repeat Yourself)